packages/terra-section-header/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Changed
+  * Renamed title prop to text to avoid conflict with native html attribute
+* Added
+  * Assigned button role to the actionable section of component
+  * aria-label and aria-expanded attribute to the actionable section of component
+
```

Comment:
```suggestion
* Changed
  * Renamed title prop to text to avoid conflict with native html attribute
  
* Added
  * Assigned button role to the actionable section of component
  * Added aria-label and aria-expanded attribute to the actionable section of component

```